packages/pwa-buildpack/envVarDefinitions.json

```diff
@@ -182,6 +182,18 @@
         }
     ],
     "changes": [
+        {
+            "name": "CHECKOUT_BRAINTREE_TOKEN",
+            "type": "defaultChanged",
+            "reason": "It was unclear that developers should provide their own Braintree tokens. An example value is provided instead",
+            "original": "sandbox_8yrzsvtm_s2bg8fs563crhqzk"
+        },
+        {
+            "name": "CHECKOUT_BRAINTREE_TOKEN",
+            "type": "exampleChanged",
+            "reason": "It was unclear that developers should provide their own Braintree tokens. An example value is provided instead",
```

Comment:
```suggestion
            "reason": "confusion about the purpose of the previously provided default value. An example is provided instead for development purposes",
```

Reply:
Fixed in https://github.com/magento/pwa-studio/pull/2008/commits/bc24b6b3182b1f33b4d39c34644acee390a231b2.